### PR TITLE
Packagecloud uploads

### PR DESCRIPTION
**Pull request by platinummonkey**
- adds signing to packages
- add ruby to so we can have packagecloud uploads
- update 'provides' for seamless upgrade to `unpoller` vs `unifi-poller`

### CODE REVIEW

File: scripts/packagecloud_push.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+gem install package_cloud
+
+ruby misc/fpm_recipe.rb
+
+PACKAGE_CLOUD_REPO="golift/pkgs"
+if [[ ${PACKAGE_VERSION} =~ .+-rc ]]; then
+  PACKAGE_CLOUD_REPO="golift/unstable"
+fi
+
+SUPPORTED_UBUNTU_VERSIONS="zesty bionic focal jammy hippo"
```
Comment (davidnewhall) [inline scripts/packagecloud_push.sh:42]: Just focal. If you push to all of these places it duplicates the package and increases storage. I've avoided it and put everything in focal.

File: scripts/packagecloud_push.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+SUPPORTED_REDHAT_VERSIONS="7 8 9"
```
Comment (davidnewhall) [inline scripts/packagecloud_push.sh:43]: same. just el7 I think is where we put these for RHEL.

File: scripts/packagecloud_push.sh (hunk 1 above)
Comment (platinummonkey) [reply scripts/packagecloud_push.sh:42]: ahhh got it

File: scripts/packagecloud_push.sh (hunk 2 above)
Comment (platinummonkey) [reply scripts/packagecloud_push.sh]: done

File: scripts/packagecloud_push.sh [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ruby misc/fpm_recipe.rb
+
+PACKAGE_CLOUD_REPO="golift/pkgs"
+if [[ ${PACKAGE_VERSION} =~ .+-rc ]]; then
+  PACKAGE_CLOUD_REPO="golift/unstable"
+fi
+
+SUPPORTED_UBUNTU_VERSIONS="focal"
+SUPPORTED_REDHAT_VERSIONS="7"
+SUPPORTED_DEBIAN_VERSIONS="buster"
```
Comment (davidnewhall) [inline scripts/packagecloud_push.sh:44]: I don't do this either. Debian can use the focal repo too. My install.sh and repo.sh scripts both point all apt users to focal.

**Review by davidnewhall — APPROVED** [review]
I dig it.

**Pull request merged**